src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.ts

```diff
@@ -95,7 +98,12 @@ export class AttachmentUnitsComponent implements OnInit {
     createAttachmentUnits(): void {
         if (this.validUnitInformation()) {
             this.isLoading = true;
-            const lectureUnitInformationDTOObj: LectureUnitInformationDTO = { units: this.units, numberOfPages: this.numberOfPages, removeBreakSlides: this.removeBreakSlides };
+            const lectureUnitInformationDTOObj: LectureUnitInformationDTO = {
+                units: this.units,
+                numberOfPages: this.numberOfPages,
+                removeBreakSlides: this.removeBreakSlides,
+                removeSolutionSlides: this.removeSolutionSlides,
+            };
```

Comment:
You can use this short version since the names are matching
```suggestion
            const lectureUnitInformationDTOObj: LectureUnitInformationDTO = { this.units, this.numberOfPages, zhis.removeBreakSlides, this.removeSolutionSlides };
```

Reply:
it gives syntax error.

Reply:
I just tried it out and you are right, the `this` seems to be causing problems.